sdk/core/azure-core/src/version.cpp

```diff
@@ -6,7 +6,7 @@
 #include <sstream>
 #include <string>
 
-using namespace Azure::Core;
+using namespace Azure::Core::Details;
```

Comment:
Add a changelog entry as this is breaking. We are effectively making version APIs "private"